src/helpers.ts

```diff
@@ -84,7 +84,7 @@ export const annouceUserIsStreaming = async (
   audioPlayer.play(audioResource);
 };
 
-export const setIntervalImmediately = (func: Function, interval: number) => {
+export const setIntervalImmediately = (func: { (): Promise<void>; (): void }, interval: number) => {
```

Comment:
I inferred the type from TypeScript, I didn't do this manually. The linter did not like generically referring to this as a Function type as it provides no type safety. I don't have much experience typing functions in TS